### DISCUSSION TITLE
where clause on a _count

### DISCUSSION THREAD

**Discussion by simonjodet** [prisma/prisma#9475] [Q&A] [answered]
Hello.

I'm trying to find orphans in a many-to-many relationship.

I tried to use _count but it doesn't seem I can use it in the where clause:
```ts
const orphans = await prisma.gift.findMany({
      where: {
        AND: [{ userId: { equals: req.user.id } }, { id: { in: giftIds } }, { _count: { contacts: { equals: 0 } } }]
      },
      include: {
        _count: {
          select: { contacts: true }
        }
      }
    });
```
I get a `Unknown arg `_count` in where` error. I tried a select instead of an include but I get the same error.

Can someone help me figure out how I can find those orphans (the gifts with 0 contact in the many-to-many relationship table)?

**Comment by ryands17** [comment, marked as answer]
@simonjodet 👋 

Currently this isn't supported by Prisma. It would be great if you could add a 👍 to this request so that we can know the priority for this one: https://github.com/prisma/prisma/issues/8413

**Reply by simonjodet** [reply]
Thanks for the answer! I've added a 👍 to #8413

**Comment by simonjodet** [comment]
For anyone wondering how I worked around this, I used this:
```
await prisma.$executeRaw(
      `DELETE FROM gifts WHERE id NOT IN (SELECT gift_contacts.giftId FROM gift_contacts)`
    );
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
